docs/src/components/community-projects.tsx

```diff
@@ -63,6 +63,16 @@ const projects: CommunityProjectProps[] = [
     description: 'Powershell Module for the Immich API',
     url: 'https://github.com/hanpq/PSImmich',
   },
+  {
+    title: 'Support mTLS with Home Assistant',
+    description: 'Allows access to Immich from public IP that did mTLS with Home Assistant.',
```

Comment:
Where is this related to HA?

Reply:
https://www.reddit.com/r/immich/comments/1e8ebbl/comment/le7v7ip/

Reply:
This is like *really* weird lol. So basically, the idea is that the home assistant app will phone home from time to time, so whenever it phones home again we whitelist that IP address, thus allowing the immich app to bypass the mTLS handshake because the IP is whitelisted.

Reply:
I really don't like the whole concept of this, and imo it doesn't add any real security benefit either. I'll let the others chime in as well though